Source/Scene/ModelExperimental/ModelSplitterPipelineStage.js

```diff
@@ -0,0 +1,65 @@
+import combine from "../../Core/combine.js";
+import ModelSplitterStageFS from "../../Shaders/ModelExperimental/ModelSplitterStageFS.js";
+import ShaderDestination from "../../Renderer/ShaderDestination.js";
+
+/**
+ * The model splitting pipeline stage is responsible for discarding fragments on the wrong side of the splitter.
+ *
+ * @namespace ModelSplitterPipelineStage
+ *
+ * @private
+ */
+const ModelSplitterPipelineStage = {};
```

Comment:
Please add a line like:

```js
ModelSplitterPipelineStage.name = "ModelSplitterPipelineStage"; // Helps with debugging
```


Since this is a namespace, not a class, it can be hard to tell them apart in the console, that's the only reason why we add this name field.